### DISCUSSION TITLE
Not able to see pandas dataframe in Interactive Data Editor using Jupyter Notebook

### DISCUSSION THREAD

**Discussion by fabianocheque** [posit-dev/positron#4166] [Q&A]
Hello everyone!

I'm testing the Positron IDE with jupyter notebook with python and I'm not able to see the pandas dataframe in the interactive data editor. 
Whe I use the the magic command `%view penguins`, I receive the following error message: **UsageError: cannot view object of type 'pandas.core.frame.DataFrame'**

![image](https://github.com/user-attachments/assets/4884ea97-9137-4038-82f4-bcf5b15d0c61)

Maybe I'm doing something wrong.

Can anyone help me this issue, pls!!

**Comment by fabianocheque** [comment]
I've restarted the kernel and preview data worked well!!

**Comment by dfalbel** [comment]
Thanks! This was probably the same issue as seen here: https://github.com/posit-dev/positron/issues/3653
And was partially addressed in https://github.com/posit-dev/positron/pull/4122
